tests/sass/property-units.sass

```diff
@@ -0,0 +1,4 @@
+.blue
+  height: 3em
+  width: 2px
+  border-top: 2px
```

Comment:
I would also second @benthemonkey here and add a lot more test cases, including multiple statement types, variable declarations, unit-less values and long form short form values too.